apps/web-giddh/src/app/app.constant.ts

```diff
@@ -634,6 +634,12 @@ export const COUNTRY_REGION_MAP: { [key: string]: string | null } = {
 /** Gst utility download portal link */
 export const GST_UTILITY_DOWNLOAD_LINK = "https://www.gst.gov.in/download/returns";
 
+/** Break Point Screen Size*/
+export const BREAKPOINT_SCREEN_SIZE = {
+    IPAD_SCREEN_SIZE: '(max-width: 1300px)',
```

Comment:
use standard screen size 
```suggestion
    SMALL_DESKTOP_SCREEN_SIZE: '(max-width: 1366px)',
```